tests/schema_external.py

```diff
@@ -24,9 +25,10 @@
     'user': 'djtest',
     'token': '2e05709792545ce'}
 
-dj.config['cache'] = {
-    'protocol': 'file',
-    'location': '/media/dimitri/ExtraDrive1/dj-store/cache'}
+cache_path = 'dj-cache'
+if not os.path.exists(cache_path):
+    os.makedirs(cache_path)
+dj.config['cache'] = cache_path
```

Comment:
Do we have explicit test cases for external store with and without using the caching? Based on this, it appears that we now only test case with caching. I think it'll make sense to have explicit tests for external storage with and without storage by switching cases.

Reply:
agreed